OasisDEX/oasis-borrow
# Piter/sc-11999/fe-make-portolio-page-static

## Changes 👷‍♀️
- Removed the following table from the portfolio view
- removed refreshing data on each block, replaced with one value pipe
- extracted portfolio page pipes to separate place and used it only for that page
  
## How to test 🧪

- open portfolio page
- must load and display all position types (maker/ajna/aave) 
- check network tabs for repeating calls to cache, all cache calls should be loaded only once on initial load (previously it was calling cache every 5 seconds repeatedly)

Know issues:
- Follow table view is left on purpose because I don't know the reasoning behind removing it, and will need to ask Frank about that. I'll create a separate ticket to track that to ensure it's cleaned up later as a CI ticket.
